aws/resource_aws_ses_email_identity.go

```diff
@@ -0,0 +1,111 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesEmailIdentity() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesEmailIdentityCreate,
+		Read:   resourceAwsSesEmailIdentityRead,
+		Delete: resourceAwsSesEmailIdentityDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				StateFunc: func(v interface{}) string {
```

Comment:
Not for this PR, but in the future we can drop the use of `StateFuncs` by adding the logic to the Read action and using a `DiffSupressFunc` to ignore emails with the trailing dots.